src/drivers/actuators/vertiq_io/module.yaml

```diff
@@ -58,7 +58,7 @@ parameters:
             type: int32
```

Comment:
Above this one line 57, it says this is sent when DISARM_TRIGGER is Set Predefined Setpoint. Shouldn't that be Set Predefined Velocity Setpoint now?

Reply:
yep, good catch.